test/integration/locations.js

```diff
@@ -261,4 +320,36 @@ describe('(/locations) Locations Interface', () => {
       })
       .catch(helpers.handler);
   });
+
+  it('DELETE /locations/countries/:uuid should delete a Country', () => {
+    return agent.delete(`/locations/countries/${country2.uuid}`)
+      .then((res) => {
+        expect(res).to.have.status(200);
```

Comment:
Go ahead and use `helpers.api.deleted(res)`.  It will make sure the API is conforming to the correct specification.